source/code-generation/query-converter/test-converted-queries/install-query-runner.txt

```diff
@@ -0,0 +1,94 @@
+.. _rm-install-query-runner:
+
+====================
+Install Query Runner
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To use Query Converter Test Bench, Relational
+Migrator must have access to the Query Runner. 
+Query Runner is a Docker container with 
+all the dependencies needed to test your converted 
+queries.
+
+About this Task
+---------------
+
+- Query Runner operates on the ``http`` protocol and 
```

Comment:
Maybe:
> Query Runner operates using the http protocol, so any requests made from a remote machine will not be encrypted. 

(Because requests made from localhost don't go out to the network so it's irrelevant).

I would drop the next sentence as it's not particularly clear of even accurate; the container connects to MongoDB where the data actually lives and sends it to whoever requested it.

Reply:
Updated.